CONTRIBUTING.rst

```diff
@@ -6,12 +6,29 @@ Where possible include updates to NEWS along with your improvements.
 
 New functionality and bug fixes should be accompanied by matching unit tests.
 
+Installing development dependencies
+-----------------------------------
+
+Install the checked out Dulwich package in editable mode with ``dev`` extras:
+
+.. code:: console
+
+   $ cd ~/path/to/checkouts/dulwich
+   # Make sure to run this in a virtual environment
+   $ pip install -e ".[dev]"
+
+This will ensure all the tools needed to test the changes are installed. Since Dulwich is
+installed in editable mode, code changes will be visible immediately, without requiring a
+reinstall (but any running Python processes will need to be reloaded to see the updated
+module definitions).
+
 Coding style
 ------------
-Where possible, please follow PEP8 with regard to coding style. Run ruff.
```

Comment:
This should just recommend running ruff without going into details.

Reply:
Not sure what you mean, which part are you referring to? If the "PEP8" part, that's just the original contents of the file, I didn't change it :). I can update it however.

Reply:
I adjusted it, does it look better to you?